src/calibrationparser.cpp

```diff
@@ -0,0 +1,45 @@
+#include "../include/calibrationparser.h"
```

Comment:
since you have "include" in the list of `include_directories` in the CMakeLists, you can just use `#include calibrationparser.h`

Reply:
wow! great!!